pennylane/ops/mid_measure.py

```diff
@@ -286,22 +495,8 @@ def circuit(x, y):
                 qml.sample(m0), qml.expval(m0), qml.var(m0), qml.probs(op=m0), qml.counts(op=m0),
             )
 
-    >>> circuit(1.0, 2.0, shots=1000)
```

Comment:
How did we miss this from the release?

Reply:
measurements weren't yet doctested.

Reply:
but I guess this is already showing the benefits of the doc testing.